### DISCUSSION TITLE
关于模板字体和页眉显示请教

### DISCUSSION THREAD

**Discussion by keykeykeykeyk** [sjtug/SJTUThesis#721] [提问] [answered]
第一个问题是关于字体，我把模板的type切换成doctor之后，生成的字体变成了仿宋。而如果使用bachelor的话，生成的字体就是宋体，请问这个可以调回宋体吗？

第二个问题是关于页眉内容的显示。学校给的模板页眉左侧是：上海交通大学学位论文，右侧为本章标题。但是这个模板默认页眉好像只有居中显示的上海交通大学学位论文，能支持显示本章标题吗？

非常感谢

**Comment by skyzh** [comment, marked as answer]
第一个问题：什么地方的字体？

第二个问题：在 \documentclass 中加入 twoside 选项

**Reply by keykeykeykeyk** [reply]
正文字体。

加了twoside ，好像并没有用
这样：
\documentclass[type=doctor, twoside]{sjtuthesis}

**Reply by skyzh** [reply]
twoside 模式下奇偶页页眉不一样，奇数为“上海交通大学学位论文”，偶数为章节名称。<del>同一行两个都显示目前不支持。</del>支持的，见下面的回复

正文字体显示为仿宋挺奇怪的。。。截个图？

**Reply by keykeykeykeyk** [reply]
![1635942488(1)](https://user-images.githubusercontent.com/12684065/140060033-4f6fd402-fdaf-4c31-851b-dc14d3ef07c2.png)
![image](https://user-images.githubusercontent.com/12684065/140060006-a0e4cdc9-06d2-49c4-ac88-41c69e92d5bf.png)

第一张的字体是仿宋了

额。而且我加了twoside之后，所有页眉都是“上海交通大学学位论文”，并不显示章节名 = =

**Reply by skyzh** [reply]
两者都是正常的宋体，`FandolSong` 不是仿宋（

怀疑你没有使用最新版。直接下载 master 上的版本试试？

**Reply by keykeykeykeyk** [reply]
哦！我去。所以FandolSong只是不同字库的宋体？我倒是用的release 1.0.0，不是master直接拉的。我过会试试好了，这应该不是问题了。麻烦大佬再帮我看下页眉的问题

**Comment by keykeykeykeyk** [comment]
刚看了下[#575](url)。问题来了怎么进else分支呢，我对latex模板的这套控制实在不熟悉
https://github.com/sjtug/SJTUThesis/blob/343c1c958461a4119ca47476a90517149a6907a0/texmf/tex/latex/sjtuthesis/sjtuthesis-graduate.ltx#L47

**Reply by skyzh** [reply]
sjtusetup.tex 中

```
  style = {
    header = double
  },
```

即可

**Reply by skyzh** [reply]
<img width="710" alt="image" src="https://user-images.githubusercontent.com/4198311/140061724-1b58ff2d-ad67-48d5-9824-dbd69906b8b2.png">

**Reply by keykeykeykeyk** [reply]
哦，确实可以了。感谢！！

刚才还去试了一波master。
在当前master分支里选用type=docter的话页眉会随着奇偶页不同，奇数为“上海交通大学学位论文”，偶数为章节名称。
另外现在doctor和bachelor生成的正文部分用adobe看的话字体都是‘宋体‘，既不是“adobe 宋”也不是“FandolSong”了
